sensors/git_activity_sensor.py

```diff
@@ -113,12 +113,14 @@ def _process_repository(self, name, repository, whitelist):
         # default value in this case rather than raise an exception.
         count = self._config['repository_sensor'].get('count', 30)
 
-        events = repository.get_events()[:count]
-        events = list(reversed(list(events)))
-
+        events = repository.get_events().reversed
```

Comment:
I think getting first 30 events from the list, and reversing those, is not the same as reversing the whole list and getting the first (last) 30 events?

That said, maybe this logic is more correct, but we should make sure

Reply:
Oh, good point.  It seems the latest returned first.  I change it.